.sync/Files.yml

```diff
@@ -348,6 +348,15 @@ group:
     repos: |
       microsoft/mu_devops
 
+# Leaf Workflow - Release Draft
+  - files:
+    - source: .sync/workflows/leaf/release-draft.yml
+      dest: .github/workflows/release-draft.yml
+      template:
+        trigger_branch_name: 'main'
+    repos: |
+      microsoft/mu_devops

```

Comment:
does it make sense to push thin into any repo we have a "main"?  

Feature repos, mu_tiano_platforms, etc?

Reply:
Yes. Honestly, I wanted to confirm everything in mu_devops for a release or two before doing so. It is tested on my fork but thought I'd double check here to reduce possible sync thrash in case any small tweaks need to be made before syncing elsewhere.